src/actions/pause/extra.js

```diff
@@ -38,21 +52,7 @@ async function getImmutableProps(expression: string, evaluate) {
 async function getExtraProps(getState, expression, result, evaluate) {
   const props = {};
   if (isReactComponent(result)) {
-    const selectedFrame = getSelectedFrame(getState());
-    const source = getSource(getState(), selectedFrame.location.sourceId);
-    const symbols = getSymbols(getState(), source);
-
-    if (symbols && symbols.classes) {
-      const originalClass = findClosestClass(symbols, selectedFrame.location);
-
-      if (originalClass) {
-        props.react = { displayName: originalClass.name };
-      }
-    }
-
-    if (!props.react) {
-      props.react = await getReactProps(evaluate);
-    }
+    props.react = await getReactProps(evaluate);
```

Comment:
@jasonLaster @mmcote Is this OK? It was the easiest way to get what I wanted but I can put it back if needed.